src/main/java/com/betterme/Book.java

```diff
@@ -0,0 +1,88 @@
+package com.betterme;
+
+import java.util.Scanner;
+
+public class Book {
+
+    private Title title;
+    private Author author;
+    private Content content;
+
+
+    public void show() {
```

Comment:
move this method after constructor pls
```suggestion
public Book(Title title, Author author, Content content) {
        this.title = title;
        this.author = author;
        this.content = content;

    }
    public void show() {
         title.show();
        author.show();
        content.show();
    }
```